### PR TITLE
Update DevKit

### PR DESCRIPTION
**Pull request by MGatner**
* Updates the CodeIgniter DevKit
* Applies latest configurations and updates

### CODE REVIEW

File: rector.php [hunk 1]
```diff
@@ -1,32 +1,121 @@
...
+        realpath(getcwd()) . '/vendor/codeigniter4/framework/system/Test/bootstrap.php',
     ]);
 
-    // is there a file you need to skip?
+    if (is_file(__DIR__ . '/phpstan.neon.dist')) {
+        $rectorConfig->phpstanConfig(__DIR__ . '/phpstan.neon.dist');
+    }
+
+    // Set the target version for refactoring
+    $rectorConfig->phpVersion(PhpVersion::PHP_74);
```
Comment (samsonasik) [inline rector.php]: this is not needed as already handled in `LevelSetList::UP_TO_PHP74`

```suggestion
```

Comment (MGatner) [reply rector.php]: I'll apply this, but this comes from the template in DevKit and should be updated there: https://github.com/codeigniter4/devkit/blob/221b22bef1b8b27ae8bf2c9d2c3ab52ad76c2660/src/Template/rector.php#L61-L62

**Comment by MGatner** [comment]
@kenjis Any idea on the Psalm error? https://github.com/codeigniter4projects/playground/runs/6574405129?check_suite_focus=true

The `require` is guarded behind `file_exists()` yet Psalm still tries to process it.

**Comment by kenjis** [comment]
@MGatner 
```diff
--- a/app/Config/Routes.php
+++ b/app/Config/Routes.php
@@ -47,5 +47,8 @@
  * needing to reload it.
  */
 if (file_exists(APPPATH . 'Config/' . ENVIRONMENT . '/Routes.php')) {
+    /**
+     * @psalm-suppress MissingFile
+     */
     require APPPATH . 'Config/' . ENVIRONMENT . '/Routes.php';
 }
```

**Pull request merged**